docs/reference/treyresearch/README.md

```diff
@@ -2,18 +2,91 @@
+# Deploy Enterprise-scale for small enterprises
+
+The Enterprise-Scale architecture is modular by design. It allows organizations to start with foundational landing zones, regardless of whether the applications are being migrated or are newly developed and deployed to Azure. The architecture enables organizations to start as small as needed and scale alongside their business requirements regardless of scale point.
 
 ## Customer profile
 
-*coming soon*
+This reference implementation provides a design path and initial technical state for Small and Medium Enterprises Azure environment based on Azure Landing Zones Design Recommendations.
+
+This architecture focuses on simplicity and provides a landing zone where a small team can confidently deploy and manage workloads.
+
+## How to evolve later
+
+If the business requirements change over time, the architecture allows for creating additional subscriptions and placing them into the suitable management group and assigning Azure policies. For more details, see the next steps section at the end of this document.
 
 ## Pre-requisites
 
-To deploy this ARM template, your user/service principal must have Owner permission at the Tenant root.
-See the following [instructions](https://docs.microsoft.com/en-us/azure/role-based-access-control/elevate-access-global-admin) on how to grant access.
+To deploy this ARM template, your user/service principal must have Owner permission at the Azure Active Directory Tenant root. See the following [instructions](https://docs.microsoft.com/en-us/azure/role-based-access-control/elevate-access-global-admin) on how to grant access before you proceed.
+
+## Optional pre-requisites
+
+The deployment experience in Azure portal allows you to bring in an existing (preferably empty) subscription dedicated to host your Platform (Management, Connectivity and Identity) resources. It also allows you to bring existing subscriptions that can be used as the initial landing zones for your applications.
+
+To learn how to create new subscriptions programmatically, please visit [Microsoft Docs](https://docs.microsoft.com/en-us/azure/cost-management-billing/manage/programmatically-create-subscription).
+
+To learn how to create new subscriptions using the Azure portal, please visit [Microsoft Docs](https://azure.microsoft.com/en-us/blog/create-enterprise-subscription-experience-in-azure-portal-public-preview/). 
 
 ## What will be deployed?
 
-*coming soon*
+By default, all recommendations are enabled. You must explicitly disable them if you do not want them to be deployed and configured.
+
+- A scalable Management Group hierarchy aligned to core platform capabilities, allowing you to operationalize at scale using centrally managed Azure RBAC and Azure Policy where platform and workloads have clear separation.
+- An Azure subscription dedicated for management, connectivity, and identity. This subscription hosts core platform capabilities such as:  
```

Comment:
It would also be great to call out here that if separating the platform subcsriptions is required due to segregation of duties and scale, we should point to Wingtip as a reference

Reply:
Is it Ok if we reference Adventure Works instead of WingTips? Think it may be more applicable as it includes the same Platform components.

Reply:
I think we should do both then, Adventure works and Wingtip, and call out the differences per the requirements. We may also surface this and make it clearer elsewhere in the RI repo, but good to have a reference here as well.